### PR TITLE
Italian and German Translations

### PR DESCRIPTION
**Pull request by djbusstop**

### CODE REVIEW

File: src/locale/es.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "housing-action-title": "Movilización para la defensa de la vivienda",
     "housing-action-description": "Donde <strong class='color-rent-strike'>acciones para la justicia</strong> están ocurriendo o han ocurrido.",
     "housing-action-form": "Complete para presentar la información sobre: Organización, resistencia y ayuda mutua en forma de huelgas de alquiler, recuperación de viviendas, ocupaciones y más. También llene si busca las redes de solidaridad y recursos para ayudar a cancelar el alquiler.",
+    "submit": "Envíe datos",
     "resources": {
       "title": "Recursos de apoyo",
       "description": "Aquí hay algunos recursos para obtener ayuda si usted está en necesidad.",
```
Comment (unixjazz) [inline src/locale/es.json:28]: line 28 with key "description" is no longer needed, it must be deleted

File: src/locale/pt-BR.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "housing-action-title": "Mobilização pelo Acesso à Moradia",
     "housing-action-description": "Onde <strong class='color-rent-strike'>ações de acesso à moradia</strong> estão ocorrendo ou ocorreram.",
     "housing-action-form": "Envie informações sobre ações pelo acesso à moradia de sua região (como ocupações, suspensão de alugueis, etc.)",
+    "submit": "Envie dados:",
     "resources": {
       "title": "",
       "description": "",
```
Comment (unixjazz) [inline src/locale/pt-BR.json:28]: line 28 is no longer needed, it must be deleted.

Comment (djbusstop) [reply src/locale/pt-BR.json]: Fixed!

File: src/locale/es.json (hunk 1 above)
Comment (djbusstop) [reply src/locale/es.json]: Fixed!

**Pull request merged**